WordPress/gutenberg
# Colorize template parts and  Reusable blocks

## What?
Adds color to template parts and reusable blocks in list view, block toolbar, and block selection outlines

## Why?
To help differentiate these block types when editing. 
Fixes: #32163

## How?
A few extra classes, a bit of CSS, and Bob's your uncle

## Todo

- [x] Combine the template part and reusable block colors
- [x] Remove color from the block name and just color the icons
- [x] Color the focus outline in the list view
- [x] Color the icon of template and reusable block children
- [x] Color the grouping background in list view
- [x] Color the hover overlay
- [x] Color the template part name in the site editor Top Bar
- [x] Color the icon in the right inspector panel heading

## Testing Instructions

- Open the list view and make sure the template parts  are colored purple in the list view and in the editor view (block toolbar title and selection outline), and that any children are also covered
- Add a reusable block somewhere in content area and make sure this also appears with the purple color 
- Also check that template parts and reusable blocks are colorized in the block inserter
- Check that the hover color of reusbale blocks in the editor is purple

## Screenshots or screencast 

https://user-images.githubusercontent.com/3629020/202564625-771e8781-4a96-4a23-be35-abbc64dbabfb.mp4
